tests/stdlib/tasynchttpserver_transferencoding.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   matrix: "--gc:arc --threads:on; --gc:arc --threads:on -d:danger; --threads:on"
+  disabled: "freebsd"
```

Comment:
unrelated but how about doing this change also: https://github.com/nim-lang/Nim/pull/18052#discussion_r641919346